main.cpp

```diff
@@ -1010,9 +891,273 @@ Part 1e - Step 19: Request a review
+    double numMilesTravelPerDay = 268.6;
+    //ticket price
+    double ticketPrice = 12;
+
+    //3 things it can do:
+    //take customer to destination
+    void takeCustomerToDestination(bool isCustomerDestination);
+
+    //check ticket
+    void checkTicket(bool customerBoughtTicket);
+
+    //pickup customers at station
+    bool pickupCustomer(int amountCustomerAtStop); //returns true if there is any customers to pick up
+};
+
+struct Restaurant 
+{
+    //number of food items
+    int numFoodItems = 15;
+    //number of plates
+    int numPlates = 200;
+    //number of chairs
+    int numChairs = 50;
+    //number of tables
+    int numTables = 15;
+    //operating hours
+    double operatingHours = 9.5;
+
+    //3 things it can do:
+    //feed customer
+    void feedCustomer(bool customerIsHungry, int numCustomerAtTable, bool foodOrderedAvailable);
+
+    //serve customer
+    void serveCustomer(bool customerOrderedFood);
+
+    //reserve seats for customer
+    bool reserveSeatsCustomer(int numCustomer, bool fullyBooked); //returns true is the amount of customers wanting to book can book
+};
+
+struct Airport
+{
+    //number of planes
+    int numPlanes = 30;
+    //lenght of runway in m
+    int lengthRunwayM = 2800;
+    //number of gates
+    int numGates = 20;
+    //secirity line wait time
+    int securityLineWaitTime = 22;
+    //number of private jets
+    int numPrivateJets = 2;
+    
+    struct CommercialPlane
+    {
+        bool isAPrivateJet = false;
+        int numEngines = 4;
+        std::string manufacturer = "Boeing";
+        std::string model = "747";
+        int numSeats = 416;
+
+        void flightCheck(float milesTraveled, int yearsOld, bool warningLightOn);
+        void restockInventory(int lenghtOfNextFlight, int stockLevelInPlane);
+        void flightOnTime(bool allPassangersAtGate, double timeArrived, double timeForDeparture);
+    };
+
+    //3 things it can do:    
+    //board customer on a plane
+    void boardCustomerOnPlane(CommercialPlane atGate);
+
+    //move luggage
+    void moveLuggage(bool conveyorBeltOn, double weightOfLuggage);
+
+    //delay flight
+    void delayFlight(double amountDelayed);
+};
+
+struct Gym 
+{
+    //number of staff
+    int numStaff = 8;
+    //number of weights
+    int numWeights = 120;
+    //number of lockers
+    int numLockers = 60;
+    //amount of cardio machines
+    int numCardioMachines = 10;
+    //brand of equipment
+    std::string brandEquipment = "Technogym";
+    //3 things it can do:
+   
+    struct BenchPress 
+    {
+        int weightBar = 20;
+        int weightPutOn = 120;
+        bool weightLocksUsed = false;
+        bool spotterInPlace = false;
+        std::string person = "bodyBuilder";
+
+        void amountReps(float weightOfPerson = 90, bool firstExercis = true);
+        void injury(std::string injuryArea, bool rippedMusle = false);
+        void lengthUsed(std::string trainingSplit = "pushPullLegs", bool trainingAlone = true);
```

Comment:
Function names reflect actions.
These function names do not reflect actions.
The VERB should always be the first word in the function name.
- [ ] revise